index.js

```diff
@@ -1,4 +1,6 @@
 export const Rels = {
+	// rels with no domain (IANA)
+	self: 'self',
```

Comment:
I'm not sure about this... would it be better just to use the 'self' string rather than adding this to hypermedia constants? I could go either way